src/cartesi-machine.lua

```diff
@@ -341,6 +354,25 @@ where options are:
     suppress cartesi-machine.lua output.
     exceptions: --initial-hash, --final-hash and text emitted from the target.
 
+  --no-init-splash
+    don't show cartesi machine splash on boot.
+
+  --append-init=<string>
+    append a command to machine's init script to be executed with root privilege,
+    the command is executed on boot after mounting flash drives and before running the entrypoint,
+    you can pass this option multiple times.
+
+  --append-init-script=<filename>
```

Comment:
Can we change "script" to "from-file"?

Reply:
Ok, but I changed to just `--append-init-file`, so without "from", because very long options are not user friendly.